strings.txt

```diff
@@ -17810,3 +17810,601 @@
+    uk = Зміни, що ви запропонували, відправлено
+    vi = Những thay đổi đề nghị của bạn đã được gửi
+    hu = Javasolt változtatásainak elküldése sikerült
+    de = Ihre Änderungsvorschläge wurden gesendet
+    fi = Ehdottamasi muutokset on lähetetty
+    cs = Odeslali jsme Vámi navržené změny
+    it = Le modifiche suggerite sono state inviate
+    nb = De foreslåtte endringene er sendt
+    zh-Hans = 您的建议已发送
+    th = คำแนะนำการเปลี่ยนแปลงของคุณถูกส่งไปแล้ว
+    ja = あなたの提案の変更が送信されました
+    ro = Modificările sugerate au fost trimise
+    ar = لقد تم إرسال التغييرات التي اقترحتها
+    sk = Vykonali sa vami odporúčané zmeny
+    es = Se han enviado sus cambios sugeridos
+    pl = Twoje sugestie zostały wysłane
+    nl = Uw voorgestelde wijzigingen zijn verzonden
+
+  [editor_comment_hint]
+    en = Comment...
+    ru = Коментарий...
+    fr = Commentaire…
+    da = Kommentar...
+    id = Komentar…
+    sv = kommentar…
+    tr = Yorum...
+    uk = Коментар…
+    vi = Nhận xét...
+    hu = Megjegyzés
+    de = Kommentar...
+    fi = Kommentti…
+    cs = Poznámka...
+    it = Commenta…
+    nb = Kommentar...
+    zh-Hans = 备注...
+    th = ข้อคิดเห็น
+    ja = コメント...
+    ro = Comentariu...
+    ar = تعليق...
+    sk = Poznámka...
+    es = Comentario…
+    pl = Komentarz…
+    nl = Reactie...
+
+  [editor_reset_edits_message]
+    en = Reset all local changes?
+    ru = Сбросить все локальные правки?
+    fr = Réinitialiser toutes les modifications locales ?
+    da = Nulstil alle lokale ændringer?
+    id = Atur ulang perubahan lokal?
+    ko = 지역 변경 사항을 재설정하시겠습니까?
+    sv = Återställ alla lokala ändringar?
+    tr = Tüm yerel değişiklikler sıfırlansın mı?
+    uk = Скинути усі локальні виправлення?
+    vi = Cài đặt lại tất cả thay đổi cục bộ?
+    hu = Elveti az összes helyi módosítást?
+    de = Alle lokalen Korrekturen verwerfen
+    fi = Nollataanko kaikki paikalliset muutokset?
+    cs = Vymazat všechny místní změny?
+    it = Reimpostare tutte le modifiche locali?
+    nb = Nullstille alle lokale endringer?
+    zh-Hans = 重置所有本地更改？
+    th = ตั้งค่าการเปลี่ยนแปลงท้องถิ่นทั้งหมด
+    ja = すべてのローカルの変更をリセットしますか？
+    ro = Resetați toate modificările locale?
+    ar = هل تريد مسح كافة التغييرات المحلية؟
+    sk = Resetovať všetky miestne časy?
+    es = ¿Restablecer todos los cambios locales?
+    pl = Skasować wszystkie lokalne zmiany?
+    nl = Alle lokale wijzigingen herstellen?
+
+  [editor_reset_edits_button]
+    en = Reset
+    ru = Сбросить
+    fr = Réinitialiser
+    da = Nulstil
+    id = Atur Ulang
+    ko = 재설정
+    sv = Återställ
+    tr = Sıfırla
+    uk = Скинути
+    vi = Cài đặt lại
+    hu = Elvetés
+    de = Verwerfen
+    fi = Nollaa
+    cs = Vymazat
+    it = Reimposta
+    nb = Nullstill
+    zh-Hans = 重置
+    th = ตั้งค่าใหม่
+    ja = リセット
+    ro = Resetare
+    ar = مسح
+    sk = Resetovať
+    es = Restablecer
+    pl = Skasuj
+    nl = Herstellen
+
+  [editor_remove_place_message]
+    en = Remove an added place?
```

Comment:
Игорь?

> On Apr 26, 2016, at 5:15 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61092824>:
> 
> > +    fi = Nollaa
> > +    cs = Vymazat
> > +    it = Reimposta
> > +    nb = Nullstill
> > +    zh-Hans = 重置
> > +    th = ตั้งค่าใหม่
> > +    ja = リセット
> > +    ro = Resetare
> > +    ar = مسح
> > +    sk = Resetovať
> > +    es = Restablecer
> > +    pl = Skasuj
> > +    nl = Herstellen
> > +
> > +  [editor_remove_place_message]
> > +    en = Remove an added place?
> Are you sure you want to remove the place you have added?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/454ca48abdc1a7a643a178fac869c249f100121f..9460d6ca1e36b2c17bee62d5d712319e41eb236f#r61092824>